documentation/automation.md

```diff
@@ -0,0 +1,142 @@
+| ----------------------------- | ----------------------------------------------------------------------------- | ----------------------------------------------- | ------------- |
+| `crlp_testing/`               | QA metadata for CRLPs.                                                        | via `deploy_rollup_testing`                     |               |
+| `delete/`                     | Removes default org metadata.                                                 | `deploy_dev_config_delete`                      |               |
+| `dev/`                        | Minimal configuration changes for development.                                | `deploy_dev_config`                             |               |
+| `offsetfiscal/`               | Configures a fiscal year starting in December.                                | via `config_offsetfiscal`                       |               |
+| `perf/`                       | Metadata for performance testing.                                             | via `test_performance`                          |               |
+| `platformencryption_fields/`  | Metadata to enable Platform Encryption for specific fields.                   | `platformencryption_deploy_field_configuration` |               |
+| `platformencryption_permset/` | Permission Set to allow managing Platform Encryption tenant secrets.          | `platformencryption_create_tenant_secret`       |               |
+| `qa/`                         | Metadata for QA and Robot tests.                                              | `deploy_qa_config`                              |               |
+| `qa_bdi_custom_metadata/`     | Custom Metadata suite for testing BDI. Can only be deployed to unmanaged orgs | `deploy_qa_bdi_metadata_config`                 |               |
+| `qa_namespaced/`              | Corrects Help Text BDI mappings in namespaced QA orgs.                        | via `qa_org_namespaced`                         |               |
+| `rd2_post_config/`            | Configuration to complete enablement of Enhanced Recurring Donations.         | `deploy_rd2_config`                             |               |
+| `reports/`                    | Reports and dashboards.                                                       | `deploy_reports`                                |               |
+| `trial/`                      | Trial-style experience metadata.                                              | `deploy_trial_config`                           |               |
+
+## Data Sets
+
+### Dev Org Data Set
+
+This data set includes most NPSP objects and is scaled based on 100 Contacts.
+
+| Deploy Task         | Capture Task | Delete Task        |
+| ------------------- | ------------ | ------------------ |
+| `test_data_dev_org` |              | `test_data_delete` |
+
+For managed orgs, the mapping must be manually set to `datasets/mapping-managed.yml`.
+
+### 1K Data Set
```

Comment:
I'm not really sure this flow works. When I tried it again this morning, it fails with the following error:
`Error on row 600: DUPLICATES_DETECTED:Duplicate Alert:--` while attempting to upload Contacts.

Reply:
It definitely doesn't work. (Or rather, it won't work on any org that has duplicate rules active).

I have a to-do list item to fix that by deactivating Duplicate Rules. Let me know if that needs to be higher priority.

Reply:
Ah. Is it worth trying to fix the data so that the dupe rule doesn't need to be deactivated? If I remember the CCI training at the sprint, it sounded like there is a theoretical process of:
1. Deploy the data into a scratch org (without dupe rules)
2. Fix the data
3. run the `extract_dataset` task to retrieve the updated data

Though I suspect it's a bit more complicated than that.

Reply:
That's pretty much it, honestly. I think that process _should_ work, but given that this is a 1k scaled data set, it may be more labor than it's worth - especially if nobody is actively using this data set.